<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorWindow</class>
 <widget class="QWidget" name="AdministratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdministratorWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QListWidget" name="coatTable"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sizeEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Colour:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colourEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="priceEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="quantityEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Photo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="photoEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
